pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -918,6 +926,35 @@ func (ctrl *PersistentVolumeController) updateVolumePhaseWithEvent(volume *v1.Pe
 	return newVol, nil
 }
 
+// assignDefaultStorageClass looks up all storage classes and attempts to assign a storage class to the given claim.
+// The storage class assigned is always the default one and if there is none or there are more defaults the claim
+// will not get updated.
+func (ctrl *PersistentVolumeController) assignDefaultStorageClass(claim *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaim, error) {
+	// [Unit test set 15]
```

Comment:
Putting test stuff like this here is inevitably going to get broken or disconnected - can we not?

Reply:
Sure, removing.